extide/gradle/src/org/netbeans/modules/gradle/actions/DefaultActionMapping.java

```diff
@@ -145,5 +145,10 @@ public boolean equals(Object obj) {
         }
         return Objects.equals(this.withPlugins, other.withPlugins);
     }
+
+    public static DefaultActionMapping DISABLED = new DefaultActionMapping();
```

Comment:
Wouldn't it be better to have a static final member in ActionMapping interface as a specific instance?

Reply:
Hm, originally I had it there - but then it's not that easy, since if the `ActionMapping` is to be returned from some Provider, it should probably hold that ActionMapping.getName().equals(the_id_passed_to_the_provider). A generic DISABLED with no `name` property at all should remain just an internal placeholder.
If something should be public, then a factory method `disabledAction(String name)` to have a correct instance floating around.

Reply:
Ok, reasoning approved, let it be. Do not make it public.